.travis.yml

```diff
@@ -1,52 +1,61 @@
+sudo: false
+python:
+  - 2.6
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - 3.5
+  - 3.6
 env:
-  # To generate (update) the env list run
-  # $ tox -l | sort | xargs -I ITEM echo "  - TOXENV="ITEM
-  # then manually sort the output so that the TOXENV values are in
-  # order by Python version and Django version
-  - TOXENV=flake8-py26
-  - TOXENV=flake8-py27
-  - TOXENV=flake8-py32
-  - TOXENV=flake8-py33
-  - TOXENV=flake8-py34
-  - TOXENV=flake8-py35
-  - TOXENV=pylint-py27-django111
-  - TOXENV=pylint-py36-django111
-  - TOXENV=py26-django14
-  - TOXENV=py26-django15
-  - TOXENV=py26-django16
-  - TOXENV=py27-django14
-  - TOXENV=py27-django15
-  - TOXENV=py27-django16
-  - TOXENV=py27-django17
-  - TOXENV=py27-django18
-  - TOXENV=py27-django19
-  - TOXENV=py27-django110
-  - TOXENV=py27-django111
-  - TOXENV=py32-django15
-  - TOXENV=py32-django16
-  - TOXENV=py32-django17
-  - TOXENV=py32-django18
-  - TOXENV=py33-django15
-  - TOXENV=py33-django16
-  - TOXENV=py33-django17
-  - TOXENV=py33-django18
-  - TOXENV=py34-django17
-  - TOXENV=py34-django18
-  - TOXENV=py34-django19
-  - TOXENV=py34-django110
-  - TOXENV=py34-django111
-  - TOXENV=py35-django18
-  - TOXENV=py35-django19
-  - TOXENV=py35-django110
-  - TOXENV=py35-django111
-  - TOXENV=py36-django111
+  - DJANGO=flake8
+  - DJANGO=1.4
+  - DJANGO=1.5
+  - DJANGO=1.6
+  - DJANGO=1.7
+  - DJANGO=1.8
+  - DJANGO=1.9
+  - DJANGO=1.10
+  - DJANGO=1.11
+matrix:
+  exclude:
+    # Python/Django combinations that aren't officially supported
+    # Django 1.4/1.5 don't support python 3
+    - { python: 3.2, env: DJANGO=1.4 }
+    - { python: 3.3, env: DJANGO=1.4 }
+    - { python: 3.4, env: DJANGO=1.4 }
+    - { python: 3.5, env: DJANGO=1.4 }
+    - { python: 3.6, env: DJANGO=1.4 }
+    - { python: 3.2, env: DJANGO=1.5 }
+    - { python: 3.3, env: DJANGO=1.5 }
+    - { python: 3.4, env: DJANGO=1.5 }
+    - { python: 3.5, env: DJANGO=1.5 }
+    - { python: 3.6, env: DJANGO=1.5 }
+    # Django 1.6 does not support 3.4-3.6
+    - { python: 3.4, env: DJANGO=1.6 }
+    - { python: 3.5, env: DJANGO=1.6 }
+    - { python: 3.6, env: DJANGO=1.6 }
+    # Django 1.7 does not support 2.6, 3.5-3.6
+    - { python: 2.6, env: DJANGO=1.7 }
+    - { python: 3.5, env: DJANGO=1.7 }
+    - { python: 3.6, env: DJANGO=1.7 }
+    # Django 1.8 does not support 2.6, 3.6
+    - { python: 2.6, env: DJANGO=1.8 }
+    - { python: 3.6, env: DJANGO=1.8 }
+    # Django 1.9/1.10 do not support 2.6, 3.2-3.3, 3.6
+    - { python: 2.6, env: DJANGO=1.9 }
+    - { python: 3.2, env: DJANGO=1.9 }
+    - { python: 3.3, env: DJANGO=1.9 }
+    - { python: 3.6, env: DJANGO=1.9 }
+    - { python: 2.6, env: DJANGO=1.10 }
+    - { python: 3.2, env: DJANGO=1.10 }
+    - { python: 3.3, env: DJANGO=1.10 }
+    - { python: 3.6, env: DJANGO=1.10 }
+    # Django 1.11 does not support 2.6, 3.2-3.3
+    - { python: 2.6, env: DJANGO=1.11 }
+    - { python: 3.2, env: DJANGO=1.11 }
+    - { python: 3.3, env: DJANGO=1.11 }
```

Comment:
Add an  `include:` here to include flake8 and Pylint in the matrix separately, independently from the Python versions that build up the matrix. See [example](https://github.com/jcassee/django-analytical/blob/master/.travis.yml#L28-L30).